wordpress-mobile/WordPress-Android
# Jetpack focus: Add tracks to banners and badges

This PR adds tracks to banners and badges and disables full screen bottom sheet for now.

Fixes #16979 

| Banner | Badge | Bottom Sheet |
|---|---|---|
|![banner_sharing](https://user-images.githubusercontent.com/4588074/178938047-3a572e9d-32a5-4a87-8885-631b1c8935dd.png)| ![home_badge_light](https://user-images.githubusercontent.com/4588074/178552554-e5b70029-b86f-4e47-8ffd-74cccb284afc.png)|![Screenshot_20220726_124506](https://user-images.githubusercontent.com/990349/180912546-399bd576-12e1-400c-8190-164757936506.png)|

To test:

- Build and run
- Launch WordPress app
- Go to App Settings (Tap on Avatar at the top right hand corner on My Site -> Find App Settings)
- Select Debug Settings
- Find JetpackPoweredBottomFeatureConfig under Features in development Re-launch app (scroll down if necessary)
- Navigate to the following screens

Find and tap on **Jetpack powered banner**. Ensure a bottom sheet pops up. Filter Logcat in AS for WordPress-STATS and observe for tracks like so 🔵 Tracked: **jetpack_powered_banner_tapped**  with respective **screen** property.
- Stats
- Notifications
- Reader
- Reader Search
- Activity Log
- Sharing

Find and tap on **Jetpack powered badge**. Ensure a bottom sheet pops up. Filter Logcat in AS for WordPress-STATS: and observe for tracks like so 🔵 Tracked: **jetpack_powered_badge_tapped** with respective **screen** property.

- Home 
- Activity Log detail
- Reader detail
- App Settings
- Notifications Settings
- Me

Tap on Get the new Jetpack app button on any bottom sheet.  Observe for tracks in Logcat
🔵 Tracked: jetpack_powered_bottom_sheet_get_jetpack_app_tapped

NOTE:  You may find an additional property like **"default_tab_experiment":"site_menu"**, that is added by another feature.

## Regression Notes
1. Potential unintended areas of impact
None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual and unit tests

3. What automated tests I added (or what prevented me from doing so)
Existing unit tests

PR submission checklist:
